collections/associate_with.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Builds a new Record using the given array as keys and choosing a value for each
+ * key using the given selector.
+ *
+ * Example:
+ *
+ * ```ts
+ * import { associateWith } from "./associate_with.ts"
+ * import { assertEquals } from "../testing/asserts.ts";
+ *
+ * const names = [ 'Kim', 'Lara', 'Jonathan' ]
+ * const namesToLength = associateWith(names, it => it.length)
+ *
+ * assertEquals(namesToLength, {
+ *   'Kim': 3,
+ *   'Lara': 4,
+ *   'Jonathan': 8,
+ * })
+ *
```

Comment:
That seems to be an accidental additional empty line

Reply:
Fixed